packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -148,45 +106,34 @@ const AccountTransferNote = ({
             <React.Fragment>
                 <AccountTransferBullet>
                     <Localize
-                        i18n_default_text='You may transfer between your Deriv fiat, cryptocurrency, and {{platform_name_mt5}} accounts.'
-                        values={{ platform_name_mt5 }}
-                    />
-                </AccountTransferBullet>
-                <AccountTransferBullet>
-                    <Localize
-                        i18n_default_text='Each day, you can make up to {{ allowed_internal }} transfers between your Deriv accounts and up to {{ allowed_mt5 }} transfers between your Deriv and {{platform_name_mt5}} accounts.'
+                        i18n_default_text='Each day you can transfer up to {{ allowed_internal }} {{ currency }}. The daily limit will be reset at 00:00 GMT.'
                         values={{
-                            allowed_internal: allowed_transfers_count?.internal,
-                            allowed_mt5: allowed_transfers_count?.mt5,
-                            platform_name_mt5,
+                            allowed_internal: (
+                                exchange_rate * (Number(allowed_transfers_amount?.internal) || 100000)
```

Comment:
```suggestion
                                 (Number(allowed_transfers_amount?.internal) || 100000) / exchange_rate
```
not sure 🤔  can u do the calculation manually and compare with this output?

Reply:
Doing this will divide the transfer amount with the exchange rate and invert the values:
Received:
<img width="275" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/edce6f95-1c2c-4554-a57d-b323a0e5bc2a">
Expected:
<img width="275" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/993e376a-be3e-400a-bc38-8a00e2ca94e4">

It might work if we switch the positions of the currency in `handleSubscription` but I don't think we have to as the position of `base_currency` and `target_currency` are already correct. Are there any pros in doing it this way?

Reply:
ok then. I was just wanted to make sure. thanks for checking